index.html

```diff
@@ -4,19 +4,19 @@
         <meta charset="UTF-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Portfolio</title>
+        <title>Abidemi's Portfolio Site</title>
         <link rel="stylesheet" href="style.css">
         <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
     </head>
-    <body>
+    <body class="scroll" id="body">
 
         <!--Header-->
         
         <header class="header">
-            <a href="#" class="logo">Abidemi</a>
+            <a href="#" class="logo" id="my_name">Abidemi</a>
             <nav class="menu-icon">
-            <i class="fas fa-bars"></i>
+                <i class="fas fa-bars"></i>
             </nav>
```

Comment:
### Your Burger Icon
![burger-icon (copy)](https://user-images.githubusercontent.com/93039359/189081127-d0c3e05a-add0-445e-96dd-6fdd673465af.png)

### Figma Burger Icon
![Screenshot from 2022-09-08 14-01-09](https://user-images.githubusercontent.com/93039359/189081449-8861ad8d-8275-4724-a654-88d7cf150c67.png)


- [ ] Please increase the size of your burger icon to make it look similar to the Figma template. :blush: